src/docker.js

```diff
@@ -113,8 +114,8 @@ exports.getHostAddress = async () => {
     const { stdout: ifconfig } = await executeContainerCommand({ container, command: ['ifconfig', defaultInterface] });
     return getInterfaceAddress(ifconfig.toString('utf8'));
   } finally {
-    // Don't wait for the container to stop (this can take a while).
-    container.stop();
+    await container.stop();
```

Comment:
Is there a reason that we are now waiting for the container to stop?

Reply:
I had mentioned this in my original PR, but I found that if we don't wait here, then the command is never sent and the container stays up after the tests complete.  There were a couple places that I identified as leaving containers running after all tests finished.